server/stt/helpers/template_helpers.py

```diff
@@ -0,0 +1,17 @@
+"""Shared template helper functions for STT."""
+
+from __future__ import annotations
```

Comment:
seems like this is [deprecated](https://docs.python.org/3/library/__future__.html#id1)